aws-cli/bash-linux/s3/bucket-lifecycle-operations/general.sh

```diff
@@ -0,0 +1,100 @@
+# No 'shebang' (#!) because this script is intended to 'source', not run.
+###############################################################################
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# This file is licensed under the Apache License, Version 2.0 (the "License").
+#
+# You may not use this file except in compliance with the License. A copy of
+# the License is located at http://aws.amazon.com/apache2.0/.
+#
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+###############################################################################
+
+# This script contains general purpose functions that are used throughout 
```

Comment:
Edit: "This script contains general-purpose functions..."

Reply:
Fixed.